src/queries/makeQueries.ts

```diff
@@ -151,7 +151,7 @@ export function makeQueries<Predicate, Options>(
   }
 
   function findByQuery(instance: ReactTestInstance) {
-    return function findFn(
+    return async function findFn(
```

Comment:
Is this change needed? Does it affects the inferred typing? The inner function does not use `await`.

If the goal here was to clarify that the function returns `Promise` then explicity typing of return type would be more readable.

Reply:
Nope it's not needed I forgot to revert this change

Reply:
I removed this